authors/apps/authentication/renderers.py

```diff
@@ -18,6 +19,11 @@ def render(self, data, media_type=None, renderer_context=None):
             # rendering errors.
             return super(UserJSONRenderer, self).render(data)
```

Comment:
in python 3, there is a better way to call super. Check it out and update this part of the code.

Reply:
😄  🆗